lib/client.c

```diff
@@ -17,6 +17,7 @@
 #include "results.h"
 #include "error.h"
 #include "chunk_downloader.h"
+#include "regex.h"
```

Comment:
Is this part of standard library? then 
```
#include <regex.h>
```

Reply:
The build script didn't catch the error. Fixing build script now.
https://ci.appveyor.com/project/smtakeda/libsnowflakeclient/build/70
```
 C:\projects\libsnowflakeclient\lib\client.c(20): fatal error C1083: Cannot open include file: 'regex.h': No such file or directory [C:\projects\libsnowflakeclient\cmake-build\snowflakeclient.vcxproj]
```